### ISSUE TITLE
From your env: `GOCACHE=off` <-- this is the problem. Unset the GOCACHE variable to let Go use the cache, it's no longer optional.

### ISSUE THREAD

**Issue by yuandongx** [golang/go#42998]
From your env: `GOCACHE=off` <-- this is the problem. Unset the GOCACHE variable to let Go use the cache, it's no longer optional.

_Originally posted by @ALTree in https://github.com/golang/go/issues/42993#issuecomment-738645702_

**Comment by yuandongx** [comment]
Continue the issue,plz!

**Comment by mvdan** [comment]
The Go project doesn't use the issue tracker for questions. See https://golang.org/wiki/Questions.

Please don't open more issues here unless you think you've found a bug. And if so, use the bug template.

If you think the original issue was closed in error, you can comment there. It's only been two hours, so opening a new issue is not necessary.
